frontend/src/scenes/settings/SettingsMap.tsx

```diff
@@ -429,6 +431,13 @@ export const SETTINGS_MAP: SettingSection[] = [
         id: 'project-danger-zone',
         title: 'Danger zone',
         settings: [
+            {
+                id: 'project-move',
```

Comment:
@Twixes the reason I added this for env as well is that if you dont have the environments flag enabled, these setttings don't show up... Can you fix it? I couldn't figure out what the logic was supposed to be...